src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -107,10 +109,78 @@ public void Start()
                     UseShellExecute = false,
                     FileName = ExePath,
                     Arguments = GetCmdArgs(port),
-                    CreateNoWindow = true
+                    CreateNoWindow = true,
+                    WindowStyle = ProcessWindowStyle.Hidden,
+                    RedirectStandardOutput = true
+                }
+            };
+            CreatePane(new Guid(), "SonarLint Daemon", true, false);
+            process.OutputDataReceived += (sender, args) =>
+            {
+                string data = args.Data;
+                if (data != null)
+                {
+                    if (data.Contains("Server started"))
+                    {
+                        CreateChannelAndStremLogs();
+                    }
+                    WritelnToPane(data);
                 }
             };
+            WritelnToPane($"Running {ExePath}");
             process.Start();
+            process.BeginOutputReadLine();
+        }
+
+        private void CreateChannelAndStremLogs()
+        {
+            channel = new Channel($"{DAEMON_HOST}:{port}", ChannelCredentials.Insecure);
+            daemonClient = new StandaloneSonarLint.StandaloneSonarLintClient(channel);
+            ListenForLogs();
+        }
+
+        private async System.Threading.Tasks.Task ListenForLogs()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Change return type to 'void'; not a single caller uses the returned value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS3241)